v2/testdata/devices.json

```diff
@@ -42,7 +42,39 @@
       "nodeKey": "nodekey:30dc3c061ac8b33fdc6d88a4a67b053b01b56930d78cae0cf7a164411d424c0d",
       "os": "linux",
       "updateAvailable": false,
-      "user": "foo@example.com"
+      "user": "foo@example.com",
+      "enabledRoutes": [
```

Comment:
Yoinked from the examples in the openapi spec
https://tailscale.com/api#tag/devices/GET/tailnet/{tailnet}/devices